backend/main.py

```
@@ -1381,93 +1389,83 @@ def check_whisper_loaded(model_size: str):
-                        # Also check for .incomplete files in blobs directory (downloads in progress)
-                        has_incomplete = False
+                        if has_model_weights:
+                            break
+
+                    has_incomplete = False
+                    try:
+                        cache_dir_path = hf_constants.HF_HUB_CACHE
+                        blobs_dir = Path(cache_dir_path) / ("models--" + hf_repo_id.replace("/", "--")) / "blobs"
+                        if blobs_dir.exists():
+                            has_incomplete = any(blobs_dir.glob("*.incomplete"))
+                    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)